.github/workflows/release.yaml

```diff
@@ -61,7 +61,7 @@ jobs:
         git config --list --global # For debug purposes.
 
     - name: Create changelog and bump
-      run: cz bump --changelog --yes
+      run: cz --no-raise 21 bump --changelog --yes
```

Comment:
@jenstroeger Can you please add a comment that:

- Commitizen should not fail when the changelog doesn't have a commit that bumps a version 
- "NoneIncrementExit == 21"
- reference to the Commitizen's [documentation](https://commitizen-tools.github.io/commitizen/bump/#avoid-raising-errors)

Reply:
Commit https://github.com/jenstroeger/python-package-template/pull/428/commits/7ec5434c1ec847eb87711a203b680a77532ff91d.